test/controllers/request_history_controller_test.exs

```diff
@@ -0,0 +1,36 @@
+defmodule Imposter.RequestHistoryControllerTest do
```

Comment:
Check your indenting across all your files, should be consistent 2 spaces indenting. Your editor should be able to reformat for you.